backend/python/autogptq/autogptq.yml

```diff
@@ -24,12 +24,12 @@ dependencies:
   - xz=5.4.2=h5eee18b_0
   - zlib=1.2.13=h5eee18b_0
   - pip:
-      - accelerate==0.23.0
+      - accelerate==0.27.0
```

Comment:
the autogptq backend uses the same environment as transformers - so the changes to the conda environment should go here: https://github.com/mudler/LocalAI/tree/master/backend/python/common-env/transformers 

however, if you find that the two can't co-exist in the same env, we might want to switch back to use a special environment for autogptq

Reply:
alright, I just found that `Makefile` actually builds `common-env/transformers`, seems the `autogptq.yml` is redundant here. Can I remove it?

Reply:
yes it is a leftover, feel free to remove it. Alternatively we can add a comment on top and warn that we keep it for reference (especially if for example the two environments can't be mixed anymore for some incompat. dependency)

Reply:
Updated, please take a look.
I updated auto-gptq needed libs only in `transformers-rocm.yml`, but I didn't test it because I don't have AMD GPU to do so. I can revert it back if you think that's not right.